anta/runner.py

```diff
@@ -44,7 +56,7 @@ def log_cache_statistics(devices: list[AntaDevice]) -> None:
             logger.info("Caching is not enabled on %s", device.name)
 
 
-async def main(  # noqa: PLR0913
+async def main(  # noqa: PLR0912 PLR0913
```

Comment:
Hmm maybe in pyproject.toml for these? Can we say what we ignore otherwise?
```suggestion
async def main(  # noqa: PLR0912 PLR0913
```